src/openms/source/FORMAT/ChromeleonFile.cpp

```diff
@@ -113,11 +113,11 @@ namespace OpenMS
     while (!ifs.eof())
     {
       TextFile::getLine(ifs, line);
-      double rt, intensity;
-      int ret = std::sscanf(line.c_str(), "%lf\t%*s\t%lf", &rt, &intensity);
+      char rt[128], intensity[128];
+      int ret = std::sscanf(line.c_str(), "%s\t%*s\t%s", rt, intensity);
```

Comment:
since the buffer is limited to 128, you should use `%128s\t%*s\t%128s` to avoid buffer overflows.